Sources/UberAuth/Client.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
+//
+
+
+import Foundation
+
+public protocol ClientProvider {
```

Comment:
Why do we need this protocol? Seems that this is should be just a struct type that carries some simple information.

Reply:
Removed